src/jsonLogic.js

```diff
@@ -107,32 +109,106 @@ export function yupSchemaWithCustomJSONLogic({ field, validations, config, id })
     );
 }
 
+function replaceHandlebarsTemplates({
+  value: toReplace,
+  validations,
+  formValues,
+  parentID,
+  name: fieldName,
+}) {
+  if (typeof toReplace === 'string') {
+    return toReplace.replace(/\{\{([^{}]+)\}\}/g, (match, key) => {
+      return validations
+        .getScope(parentID)
+        .evaluateComputedValueRuleForField(key.trim(), formValues, fieldName);
+    });
+  } else if (typeof toReplace === 'object') {
+    const { value, ...rules } = toReplace;
+
+    const computedTemplateValue = Object.entries(rules).reduce((prev, [key, rule]) => {
+      const computedValue = validations.getScope(parentID).evaluateValidation(rule, formValues);
+      return prev.replaceAll(`{{${key}}}`, computedValue);
+    }, value);
+
+    return computedTemplateValue.replace(/\{\{([^{}]+)\}\}/g, (match, key) => {
+      return validations
+        .getScope(parentID)
+        .evaluateComputedValueRuleForField(key.trim(), formValues, fieldName);
+    });
+  }
+}
+
 export function calculateComputedAttributes(fieldParams, { parentID = 'root' } = {}) {
-  return ({ validations, formValues }) => {
+  return ({ validations, isRequired, config, formValues }) => {
     const { name, computedAttributes } = fieldParams;
     const attributes = Object.fromEntries(
       Object.entries(computedAttributes)
         .map(handleComputedAttribute(validations, formValues, parentID, name))
         .filter(([, value]) => value !== null)
     );
 
-    return attributes;
+    return {
+      ...attributes,
+      schema: buildYupSchema(
+        { ...fieldParams, ...attributes, required: isRequired },
+        config,
+        validations
+      ),
+    };
   };
 }
 
 function handleComputedAttribute(validations, formValues, parentID, name) {
   return ([key, value]) => {
+    if (key === 'description')
```

Comment:
Can we add `{}` to all the conditional blocks here for consistency, please? 🙏

Reply:
I'm surprised eslint isn't setup to enforce [multi-line](https://eslint.org/docs/latest/rules/curly#multi-line) 🤔 .

Could you add to `.eslintrc` the following rule?

```
    "curly": ["error", "multi-line"],
```

Reply:
this is done